### PR TITLE
ENTESB-4090 - fixes for kie-camel tests

### PR DESCRIPTION
**Pull request by mrietveld**
Related to https://github.com/jboss-integration/fuse-bxms-integ/pull/159

Please cherry-pick to 6.3.x

This fix is for: https://bugzilla.redhat.com/show_bug.cgi?id=1267906 , which is a blocker bug.

### CODE REVIEW

**Comment by mrietveld** [comment]
ok to test

File: drools-core/src/main/java/org/drools/core/xml/jaxb/util/ObjectFactory.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2010 JBoss Inc
```
Comment (psiroky) [inline drools-core/src/main/java/org/drools/core/xml/jaxb/util/ObjectFactory.java:2]: Nit picking...should be 2015

**Comment by mariofusco** [comment]
Merged by https://github.com/droolsjbpm/drools/commit/ce990f884

**Comment by mariofusco** [comment]
Cherry-picked to 6.3.x by https://github.com/droolsjbpm/drools/commit/382a9feb2

**Pull request closed without merging**